components/dashboard/src/workspaces/WorkspaceEntry.tsx

```diff
@@ -91,8 +86,22 @@ export function WorkspaceEntry({ desc, model, isAdmin, stopWorkspace }: Props) {
     }
     menuEntries.push({
         title: "Download",
-        href: downloadURL,
-        download: `${ws.id}.tar`,
+        customContent: (
+            <div className="">
+                <span className="block text-gray-300">Download</span>
+                <span className="text-gray-400">
+                    Deprecated.{" "}
+                    <a
+                        href="https://github.com/gitpod-io/gitpod/issues/7901"
+                        className="gp-link"
+                        target="_blank"
+                        rel="noreferrer"
+                    >
+                        Learn more
```

Comment:
This currently links to https://github.com/gitpod-io/gitpod/issues/7901 as a minimal viable change. If and when we have a section in the docs we can link to for extensive explanation we could do that. Cc @kylos101 @atduarte 

From https://github.com/gitpod-io/gitpod/issues/14364#issuecomment-1302082136:

> | Current state of #14393 |
> |-|
> | <img width="1440" alt="Screenshot 2022-11-03 at 3 06 37 PM" src="https://user-images.githubusercontent.com/120486/199728340-6dc71050-f33d-4782-a94d-a4b04d5cd842.png"> |